homeassistant/components/alarm_control_panel/device_action.py

```diff
@@ -57,7 +65,7 @@ async def async_get_actions(hass: HomeAssistant, device_id: str) -> List[dict]:
         if state is None:
             continue
 
-        supported_features = state.attributes[ATTR_SUPPORTED_FEATURES]
+        supported_features = state.attributes["supported_features"]
```

Comment:
What was the reason for this change? Shouldn't we still just use the constant provided by `homeassistant.const`?

Reply:
Removing /env/Scripts reults in build errors.. As these are not part of the PR how to solve? I'm nog an experienced git user

Reply:
Solved, in prior build this constant was triggering errors, so i removed it temporarily. Lucky you have a sharp eye on my work:-)